keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/ResourceTagger.kt

```diff
@@ -83,25 +83,53 @@ class ResourceTagger(
 
   @EventListener(ResourceCreated::class)
   fun onCreateEvent(event: ResourceCreated) {
-    if (event.resourceId.shouldTag()) {
+    if (!event.resourceId.shouldTag()) {
+      return
+    }
+    /**
+     * If a MultiRegion resource provides a set of regional resourceIds, entity tags will be created
+     * against those ids only, not the
```

Comment:
is there a missing part of this comment sentence?

Reply:
😱